caddytest/integration/caddyfile_adapt/header.txt

```diff
@@ -2,7 +2,7 @@
 	header Denis "Ritchie"
 	header +Edsger "Dijkstra"
 	header ?John "von Neumann"
-	header -Wolfram
+	header -Wolfram*
```

Comment:
This should be `.*`, no? I don't think `*` is useful there unless we want to say that `Wolframmmm` is useful to match 😂 

```suggestion
	header -Wolfram.*
```